aladdin/commands/build_components/__init__.py

```diff
@@ -0,0 +1,8 @@
+from aladdin.commands.build_components import __main__
+
+
+def parse_args(sub_parser):
+    subparser = sub_parser.add_parser(
+        "build-components", help=__main__.__doc__
+    )
+    subparser.set_defaults(func=lambda args: __main__.main())
```

Comment:
is there a reason we don't pass args to main()?  This seems slightly counterintuitive.